app/services/weets.js

```diff
@@ -0,0 +1,22 @@
+const axios = require('axios');
```

Comment:
You can create a function that wraps the axios package and call it from anywhere in the app

Reply:
Done! thanks by the feedback